mappings/net/minecraft/world/dimension/DimensionType.mapping

```diff
@@ -1,58 +1,64 @@
 CLASS net/minecraft/unmapped/C_oxijnccf net/minecraft/world/dimension/DimensionType
-	FIELD f_bcecvbdh bedWorks Z
 	FIELD f_cdpesyot REGISTRY_CODEC Lcom/mojang/serialization/Codec;
-	FIELD f_ckygwuem ambientLight F
-	FIELD f_exnsbfbw MIN_HEIGHT I
-	FIELD f_eyjygtak MAX_COLUMN_HEIGHT I
-	FIELD f_hmvxmnvw MIN_SECTION I
-	FIELD f_iilfmdxv MAX_HEIGHT I
+	FIELD f_exnsbfbw MIN_Y I
+	FIELD f_eyjygtak MAX_Y I
+	FIELD f_hmvxmnvw FAR_BELOW_MIN_Y I
```

Comment:
`FAR_BELOW_MIN_Y` isn't very clear

would something like `OUT_OF_WORLD_Y` make more sense?

or `EXTREMELY_LOW_Y`, `EXTREMELY_LOW_Y_THRESHOLD`?

Reply:
`FAR_BELOW_MIN_Y` seems to be used in `Aquifer` to disable it's generation in some places by using it as surface level, so I guess `OUT_OF_WORLD_Y` or `BELOW_WORLD_Y` would make sense